src/factories/uuid.ts

```diff
@@ -0,0 +1,9 @@
+import { TOptions } from '../types/options';
+import { TStringJSONSchema } from '../types/string-json-schema';
+import { string } from './string';
+
+const UUID_PATTERN: string = '^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$';
```

Comment:
https://github.com/bluebirds-blue-jay/schema/blob/master/src/utils/to-string-reg-exp.ts will help you with regexps